composer.json

```diff
@@ -30,7 +30,10 @@
     "autoload": {
         "psr-4": {
             "CacheWerk\\BrefLaravelBridge\\": "src"
-        }
+        },
+        "files": [
+            "src/bref-init.php"
```

Comment:
Where is this file loaded? Does it need to be auto-loaded?

Reply:
This is a way to have this file run by default (it is loaded by Composer every time), which lets us register extensions into Bref:

- lets us register hooks
- lets us register the Bref "handler resolver" (the default Bref handler resolver lets users define files as handlers, e.g. `public/index.php` -> we override the handler resolver so that Laravel users can set classes/services directly, e.g. `App\MyS3Handler`, or, in our case, the bridge SQS handler)

Kind of like Laravel service providers, except we don't need a Composer plugin or advanced system.

It's an extension API that has stayed under the radar for a while, and that I used in the Symfony Bridge last year to provide the exact same feature (https://github.com/brefphp/symfony-bridge/pull/45, docs: https://github.com/brefphp/symfony-bridge#class-handlers). We'll finally have that for Laravel too, and we'll directly make use of it for Octane and SQS handlers.